sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5553,9 +5553,10 @@ def test_issue_7180():
 
 
 def test_pretty_Complement():
-    assert pretty(S.Reals - S.Naturals) == '(-oo, oo) \ Naturals()'
+
+    assert pretty(S.Reals - S.Naturals) == 'S.Reals \ S.Naturals'
     assert upretty(S.Reals - S.Naturals) == u'ℝ \ ℕ'
-    assert pretty(S.Reals - S.Naturals0) == '(-oo, oo) \ Naturals0()'
+    assert pretty(S.Reals - S.Naturals0) == 'S.Reals \ S.Naturals'
```

Comment:
The pretty printing for reals should stay the same. Also, S.Naturals0 should not print as S.Naturals!